### ISSUE TITLE
Pairing does not work

### ISSUE THREAD

**Issue by tarquinwj** [richsmith/sexytopo#167]
Start with a DistoX2 that is not connected, and not paired.
Switch on the Disto.
In SexyTopo's connection screen, tap the "PAIR" button.
SexyTopo says "Unable to scan (bluetooth failure?)".

Android 11
SexyTopo 1.6.1

**Comment by richsmith** [comment]
I don't think this is a SexyTopo issue. If it requests a bluetooth scan and is refused, not sure what SexyTopo can do about it apart from report it. I have seen this once on requesting a scan straight after another scan; presumably the hardware was still busy.

**Comment by tarquinwj** [comment]
We tried this on the training course with 3 separate devices, and I have tried it several times with my own two Android devices. They all failed. I don't know if this is an Android version issue or what. Certainly I think it is an Android issue, rather than SexyTopo's. Wondering if there is some new Android change that has broken it forever, like they did with file permissions. Just ... wish there were a way around it, or a way to catch it and show a notice "use the system Bluetooth settings instead".

**Comment by richsmith** [comment]
Oh OK. Maybe there is a problem here then. Several people managed to pair OK on the training at CHECC, but come to think of it one person couldn't.

**Comment by CaverBruce** [comment]
Firing up my SexyTopo to help a friend debug some issues.  Same issue as Tarquin with a phone that last time (more than a year ago) worked fine.  Instant response with error message, so as though ST cannot see bluetooth at all.  Other bluetooth apps are working normally.  I turned them all off and restarted phone in case there might have been interference.  No change.
Fearing disto failure I checked with PocketTopo (on a different device) and the disto connected and data downloaded normally.
So phone seems OK and disto seems OK, but ST has an instantaneous 'bluetooth failure' response.

Android Galaxy S10+
Android 10
SexyTopo 1.6.1
Leica DistoX2  4538
